packages/trpc/server/createContext.ts

```diff
@@ -128,9 +137,12 @@ export async function createContextInner(opts: CreateInnerContextOptions) {
  * Creates context for an incoming request
  * @link https://trpc.io/docs/context
  */
-export const createContext = async ({ req, res }: CreateContextOptions, sessionGetter = getSession) => {
+export const createContext = async (
+  { req, res }: CreateContextOptions,
+  sessionGetter: GetSessionFn = DEFAULT_SESSION_GETTER
```

Comment:
Since this has a default fallback we can set it as optional

```suggestion
  sessionGetter?: GetSessionFn = DEFAULT_SESSION_GETTER
```